src/source/src/connector_source.rs

```diff
@@ -162,16 +162,7 @@ impl ConnectorSource {
                     // TODO: is this reader split across multiple threads...? Realistically, we want
                     // source_ctx to live in a single actor.
                     let source_ctx = source_ctx.clone();
-                    async move {
-                        create_split_reader(
-                            *props,
-                            splits,
-                            parser_config,
-                            source_ctx,
-                            data_gen_columns,
-                        )
-                        .await
-                    }
+                    create_split_reader(*props, splits, parser_config, source_ctx, data_gen_columns)
```

Comment:
no `await` here? I thought it is about to wait till recv handshake resp from embedded dbz

Reply:
We has await on the outer `try_join_all`